src/types.ts

```diff
@@ -28,12 +28,6 @@ export interface RouteConfig {
   skipAppWrapper?: boolean;
 }
 
-export interface SignalLike<T> {
-  value: T;
-  peek(): T;
-  subscribe(fn: (value: T) => void): () => void;
-}
```

Comment:
This is used later in the file, in some commented code. But I guess it can be re-added once the todo is resolved.

Reply:
Fair point, opened https://github.com/denoland/fresh/issues/2537 to keep track of that.